src/applications/edu-benefits/0994/analytics-functions.js

```diff
@@ -1,30 +1,30 @@
 import recordEvent from '../../../platform/monitoring/record-event';
-import _ from 'lodash';
+import { get, isEmpty, isBoolean } from 'lodash';
```

Comment:
For consistency should we do:
```js
import get from 'lodash/get';
import isEmpty from 'lodash/isEmpty';
import isBoolean from 'lodash/isBoolean';
```

Reply:
@kelsonic Yeah I went this route for 3 imports or more to keep the number of lines down.